packages/material/src/util/layout.tsx

```diff
@@ -51,32 +54,56 @@ export const renderLayoutElements = (
   ));
 };
 
+const areEqual = (
```

Comment:
Why is this re-defined here?

Reply:
yes, the types didn't match. But it would make sense to find a common type between MaterialLayoutType and JsonFormsPropTypes, or maybe extend the JsonFormsPropTypes, I can have a look at that

Reply:
Right, now that you mention it, I think this is also the case for the `ExpandPanelRenderer`. I'd be great if we could unify all of these.

Reply:
I'll merge this one then, let's create a follow-up. Could you take care of that?

Reply:
Also this needs to be rebased before I can merge it.

Reply:
rebased and removed the unnecessary areEqual